tools/test-static-code

```diff
@@ -124,7 +124,7 @@ else
     #   - executable matching ^#!/usr/bin/python3
     PYFILES="$(find test/ tools/ -type f '(' -name '*.py' -o -executable -execdir grep -q '^#!/usr/bin/python3' '{}' ';' ')'  -print)"
     # FIXME: Fix code for the warnings and re-enable them
-    out=$(python3 -m "$PYTHON_STYLE_CHECKER" --ignore E501,E261,W504 $PYFILES) || true
+    out=$(python3 -m "$PYTHON_STYLE_CHECKER" --max-line-length=415 --ignore E261,W504 $PYFILES) || true
```

Comment:
FTR, I agree to moving to `--max-line-length`, this makes more sense. We can then try to reduce this to at least 150 or so (in a follow-up)